Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/tracing/details/TracingDetailsState.kt

```diff
@@ -26,28 +29,26 @@ data class TracingDetailsState(
      * in all cases when risk level is not increased
      */
     fun isBehaviorNormalVisible(): Boolean =
-        riskLevelScore != RiskLevelConstants.INCREASED_RISK
+        riskState != INCREASED_RISK
```

Comment:
Is there a reason why we do not simply call it HIGH_RISK? (independent of the wording we use to display it to the user) As far as I understand, it's either low = green or high = red

Reply:
I had it previously named like that and @harambasicluka expressed the desire to keep it `INCREASED_RISK`.

Reply:
I wanted to have it consistently used in our app and as I think it's called increased risk in the protobuff files and since the beginning I would stick to increased risk.